piperider_cli/profiler/profiler.py

```diff
@@ -130,11 +167,13 @@ def profile(self, subjects: List[ProfileSubject] = None) -> dict:
         table_index = 0
         self.event_handler.handle_run_progress(result, table_count, table_index)
 
+        map_name_tables = await self._fetch_metadata(subjects)
```

Comment:
`_fetch_metadata` progress bar still showed up when `subjects` is an empty array